permissions/build.gradle

```diff
@@ -82,6 +82,9 @@ dependencies {
     implementation libs.kotlin.coroutines.android
     implementation libs.napier
 
+    lintChecks(project(":permissions-lint"))
```

Comment:
Do you need this?
https://developer.android.com/studio/releases/gradle-plugin?buildsystem=ndk-build#new_features_5

(you might, but not sure. if you just want consumers of the library to use those checks then lintPublish should be enough?)

Reply:
I think it's good to have, to prevent bad usages of the APIs in tests, for example